programming/Encoders.md

```diff
@@ -0,0 +1,57 @@
+<b>Ever been on a website written in another language and had to utilize google translate to gain some 
+semblance of recognition as to what it was you were looking at?</b>
+Encoders work in a similar fashion to Google Translate, of course instead of translating languages it merely translates
+given data into another format.
+
+There's lots of different examples of encoders at work all around us, such as:
+  * Elevators - Encoders in elevators tell the elevator's controller device its position so that it can properly align itself with the floor and open up the doors
+  at the correct position.
+  * Cranes - Some large cranes use encoders in order to collect accurate location information to know where it is.
+  * Computer Microphones - Encoders are used to convert those signals your microphone picks up into actual data that your computer can understand!
```

Comment:
Computer microphones use the other kind of encoding, not the one we care about here. Can you remove this example?